scriptmodules/libretrocores/lr-mame2003-plus.sh

```diff
@@ -12,21 +12,61 @@
 rp_module_id="lr-mame2003-plus"
 rp_module_desc="Arcade emu - updated MAME 0.78 port for libretro with added game support"
 rp_module_help="ROM Extension: .zip\n\nCopy your MAME roms to either $romdir/mame-libretro or\n$romdir/arcade"
-rp_module_licence="NONCOM https://raw.githubusercontent.com/libretro-mirrors/mame2003-plus-libretro/master/docs/mame.txt"
-rp_module_section="exp"
+rp_module_licence="NONCOM https://github.com/arcadez/-mame2003-plus-libretro/blob/master/docs/mame.txt"
+rp_module_section="main"
```

Comment:
this will need to stay in experimental for now.

Reply:
Ok then